### ISSUE TITLE
cozy-monitor status --raw

### ISSUE THREAD

**Issue by nledez** [cozy/cozy-monitor#81]
Hello,

I would like to have a "cozy-monitor status --raw" (or another parameter, it doesn't care) to have an output without color:
```
postfix: up
couch: up
controller: up
ds: up
home: up
proxy: up
index: up
sync: up
calendar: up
contacts: up
```

It's more easier for scripting. Thanks.

**Comment by poupotte** [comment]
It's fixed with a805fa8

**Comment by nledez** [comment]
Thanks! ❤️🌹
